forch/faucet_state_collector.py

```diff
@@ -635,19 +636,19 @@ def _fill_acls_behavior(self, switch_name, acls_map_list, acls_config,
             rules_map_list = acl_map.setdefault('rules', [])
 
             for rule_config in acl_config.rules:
-                cookie = str(rule_config.get('cookie'))
+                num_cookie = rule_config.get('cookie')
```

Comment:
cookie_num --- num_cookie would be a count of the number of cookies.

Reply:
Done